examples/data/q01-01.js

```diff
@@ -329,8 +329,8 @@ var relational_01 = relational_01a;
 
 var relational_01_neg = {
     "resultset": [
-        ["London", "2011-06-05", -72],
-        ["London", "2011-06-12", -50],
+        ["London", "2011-09-05", -72],
```

Comment:
Please don't change this dataset as it is used by many examples, and these may be depending on the current data.